flag_float64_slice.go

```diff
@@ -129,6 +129,9 @@ func (f *Float64SliceFlag) Apply(set *flag.FlagSet) error {
 				}
 			}
 
+			// Set this to false so that we reset the slice if we then set values from
+			// flags that have already been set by the environment.
+			f.Value.hasBeenSet = false
```

Comment:
Not sure if you need this. Did you try running the additional tests with s.hasBeenSet = true , they run just fine.

Reply:
I'm also not sure, but since this matches what we have for the other flags, I think it's a reasonable change to bring in.